bluepyopt/ephys/morphologies.py

```diff
@@ -57,6 +57,8 @@ def __init__(
             do_set_nseg=True,
             comment='',
             replace_axon_hoc=None,
+            axon_stump_length=60,
```

Comment:
@Hjorthmedh : Could you replace the word `stump` with `stub` to be consistent with wording at other places in Bluepyopt and Bluepyemodel? Thanks.

Reply:
Good catch, updated.

Reply:
Thank you!